src/coreclr/vm/CMakeLists.txt

```diff
@@ -759,6 +759,13 @@ else(CLR_CMAKE_TARGET_WIN32)
             ${ARCH_SOURCES_DIR}/crthelpers.S
             ${ARCH_SOURCES_DIR}/pinvokestubs.S
         )
+    elseif(CLR_CMAKE_TARGET_ARCH_LOONGARCH64)
```

Comment:
If move changes from this file to https://github.com/dotnet/runtime/pull/62885 then this PR would not have any unreferenced files lurking around and would be easier to validate VM PR did not miss any files.